src/components/moto/moto_form.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { Motorcycle } from '../../types';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import ClearIcon from '@mui/icons-material/Clear';
+import InputLabel from '@mui/material/InputLabel';
+import { MenuItem, Alert, TextField, FormControl, Select, SelectChangeEvent, Paper,Box } from '@mui/material';
+
+const initialMotorcycleState: Motorcycle = {
+  id: 0,
+  maker: '',
+  model: '2020',
+  year: 2020,
+};
+
+interface MotorcycleFormProps {
+  addMotorcycle: (motorcycle: Motorcycle) => void;
+}
+
+export default function MotorcycleForm({ addMotorcycle }: MotorcycleFormProps) {
+  const [motorcycle, setMotorcycle] = useState<Motorcycle>(initialMotorcycleState);
+  const [errors, setErrors] = useState<string[]>([]);
+
+  const validate = () => {
+    const newErrors: string[] = [];
+    if (!motorcycle.maker) {
+      newErrors.push('Make is required');
+    }
+    if (!motorcycle.model) {
+      newErrors.push('Model is required');
+    }
+    if (motorcycle.year <= 1900) {
+      newErrors.push('Year must be greater than 1900');
+    }
+    setErrors(newErrors);
+    return newErrors.length === 0;
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+    setMotorcycle({ ...motorcycle, [name]: value });
+  };
+
+  const handleSelectChange = (event: SelectChangeEvent<string>) => {
+    const { value } = event.target;
+    setMotorcycle({ ...motorcycle, maker: value });
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (validate()) {
+      addMotorcycle(motorcycle);
+      setMotorcycle(initialMotorcycleState);
+    }
+  };
+
+  const handleClear = () => {
+    setMotorcycle(initialMotorcycleState);
+    setErrors([]);
+  };
+
+  return (
+    <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100vh', p: 2 }}>
+
+    <Paper sx={{ padding: 2 }}>
+    <form onSubmit={handleSubmit}>
+        <Stack direction="column" spacing={2}>
+          <FormControl >
+            <InputLabel>Make</InputLabel>
+            <Select
+              id="make"
+              name="make"
+              label="Make"
+              defaultValue='Select Maker'
+              value={motorcycle.maker}
+              onChange={handleSelectChange}
+            >
+              <MenuItem value="">Select Maker</MenuItem>
+              <MenuItem value="Toyota">Suzuki</MenuItem>
+              <MenuItem value="Honda">Yamaha</MenuItem>
+              <MenuItem value="Ford">BMW</MenuItem>
```

Comment:
Instead of repeating the same `<MenuItem>` component , you can create an `enum` or `array` for all options you have, and then map it to return all the menu items.
For example: `makers.map((maker) => <MenuItem value={maker}>{maker}</MenuItem>

Reply:
This will also prevent you from the making mistakes when value and label are differ. 
If you pay attention you have the value `Toyota` when label is `Suzuki`, etc.